src/strings/en-us.json

```diff
@@ -955,14 +955,14 @@
     "LabelTitle": "Title",
     "LabelTonemappingAlgorithm": "Select the Tone mapping algorithm to use",
     "LabelTonemappingDesat": "Tone mapping desat",
-    "LabelTonemappingDesatHelp": "Apply desaturation for highlights that exceed this level of brightness. The higher the parameter, the more color information will be preserved. This setting helps prevent unnaturally blown-out colors for super-highlights, by (smoothly) turning into white instead. This makes images feel more natural, at the cost of reducing information about out-of-range colors. The recommended and default values are 0 and 0.5.",
+    "LabelTonemappingDesatHelp": "Apply desaturation for highlights that exceed this level of brightness. The higher the parameter, the more color information will be preserved. This setting helps prevent unnaturally blown-out colors for super-highlights, by (smoothly) turning into white instead. This makes images feel more natural, at the cost of reducing information about out-of-range colors. The recommended value is 0.",
     "LabelTonemappingParam": "Tone mapping param",
     "LabelTonemappingParamHelp": "Tune the tone mapping algorithm. The recommended and default values are NaN. Generally leave it blank.",
     "LabelTonemappingPeak": "Tone mapping peak",
-    "LabelTonemappingPeakHelp": "Override signal/nominal/reference peak with this value. Useful when the embedded peak information in display metadata is not reliable or when tone mapping from a lower range to a higher range. The recommended and default values are 100 and 0.",
+    "LabelTonemappingPeakHelp": "Override signal/nominal/reference peak with this value. Useful when the embedded peak information in display metadata is not reliable or when tone mapping from a lower range to a higher range. The recommended value is 100.",
```

Comment:
```suggestion
    "LabelTonemappingPeakHelp": "Override signal/nominal/reference peak with this value when the embedded peak information in display metadata is not reliable or when tone mapping from a lower to a higher range. The recommended value is 100.",
```

Reply:
I think this changes the meaning because the old string implies it overrides the value on all media while the suggested change implies that it is only overridden when the metadata of source media is unreliable

Reply:
```suggestion
    "LabelTonemappingPeakHelp": "Override the embedded metadata value for the input signal with this peak value instead. The default value is 100 (1000nit).",
```

Reply:
Don't explain too much because the user still won't get what value to set.